lib/index.js

```diff
@@ -19,6 +89,57 @@ function handleTokenRequest({logger}) {
     }
   };
 }
-odule.exports = {
-  handleTokenRequest
+
+function _authenticateClient({client, clientSecret}) {
+  if(!client) {
+    throw new InvalidClient({
+      description: 'Unknown client identifier.'
+    });
+  }
+
+  const storedClientSecret = client.clientSecret || client.client_secret;
```

Comment:
Why do we need to support both of these?

Reply:
I forget what my reasoning was. We probably don't need the snake case one.

Reply:
Removed.